### PR TITLE
Small fixes and improvements

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

**Comment by illwieckz** [comment]
I guess “the area displayed around eggs, overmind and reactor” was about power range but that does not exist anymore as you noticed.

File: src/cgame/cg_buildable.cpp [hunk 1]
```diff
@@ -2229,11 +2229,12 @@ void CG_Buildable( centity_t *cent )
 
 	if ( !( es->eFlags & EF_B_SPAWNED ) )
 	{
-		sfxHandle_t prebuildSound = cgs.media.humanBuildablePrebuild;
+		sfxHandle_t prebuildSound;
 
 		switch ( team )
 		{
 		case TEAM_HUMANS:
+		default:
```
Comment (illwieckz) [inline src/cgame/cg_buildable.cpp:2222]: maybe:

```
default: // make the compiler happy by making it knows prebuildSound is always set
```

or something like that (if that was the issue)?

Comment (necessarily-equal) [reply src/cgame/cg_buildable.cpp]: I've dropped the commit for now, I guess it will need proper multiple team handling anyway.

**Comment by slipher** [comment]
I would rather drop the indentation-only commits. We have thousands and thousands of places with ugly indentation so it is a waste of time to be manually fixing them piecemeal, and meanwhile it breaks blame.

Other commits look good.

**Pull request merged**